riptide-opentracing/src/main/java/org/zalando/riptide/opentracing/span/SpanOperator.java

```diff
@@ -0,0 +1,10 @@
+package org.zalando.riptide.opentracing.span;
+
+import io.opentracing.Span;
+
+import java.util.List;
+import java.util.Map;
+
+interface SpanOperator {
+    void apply(Span span, Map<String, List<String>> headers);
```

Comment:
not the best naming example here.

How `SpanOperator` relates to `headers`? Seems to me like abstraction leak.

Reply:
Totally agree. Struggled with a good design there. That includes names. Since all of this is package private I don't care that much right now, but if you have a good suggestion, I'm all for it.

Reply:
`HttpSpanOperator` as name will provide at least context for the headers parameters here.